src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private static final int MAX_VALUE = 100;
+    private static final String DEF_COLOR = "WHITE";
+    private static final int DEF_RADIUS = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();

```

Comment:
why is one final and the other isn't? Both variants are acceptable but consistency matters

Reply:
not fixed